site/content/blog/2022-08-01-whats-new-in-svelte-august-2022.md

```diff
@@ -0,0 +1,112 @@
+---
+title: "What's new in Svelte: August 2022"
+description: "Changes to SvelteKit's `load` before 1.0, better Vite configurations, plus some new HTML and CSS properties!"
+author: Daniel Sandoval
+authorURL: https://desandoval.net
+---
+
+There's lots to cover this month... with big changes are coming to SvelteKit's design before 1.0 can be completed. If you haven't already, check out Rich's Discussion, [Fixing `load`, and tightening up SvelteKit's design before 1.0 #5748](https://github.com/sveltejs/kit/discussions/5748).
```

Comment:
This seems to be a typo also
`There's a lot to cover this month...big changes are coming to SvelteKit's design before 1.0 can be completed.`